### PR TITLE
2 actvie color support

### PR DESCRIPTION
**Pull request by aleksi100**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Make intuitve pixel art in this simple drawing tool written in _vanilla javascri
 | toggle grid      | G                   |
 | undo             | Z                   |
 | redo             | X                   |
+| color swap       | C                   |
```
Comment (Kully) [inline README.md:29]: Nice job!

File: js/script.js [hunk 2]
```diff
@@ -8,8 +8,10 @@ const BUTTON_UP_OUTLINE = "";
 const BUTTON_DOWN_COLOR = "#f0f0f0";
 const BUTTON_DOWN_OUTLINE = "1px solid blue";
 const CANVAS_INIT_COLOR = palette_color_array[0];
+let ACTIVE_COLOR_SELECT = "1nd-Color";
 const STATE = {
-    "activeColor": palette_color_array[palette_color_array.length - 1],
+    "1nd-Color": palette_color_array[palette_color_array.length - 1],
```
Comment (Kully) [inline js/script.js:13]: Small nitpick, but can you rename `1nd-Color` and `2nd-Color` to `firstColor` and `secondColor` respectively? It's best to keep the camelCase consistent in at least this STATE object.

File: js/script.js [hunk 3]
```diff
@@ -143,13 +147,33 @@ function Alert_User(text)
...
+{
+    let activeColorDiv_1 = document.getElementById("active-color-preview-1");
+    let activeColorDiv_2 = document.getElementById("active-color-preview-2");
+    if(ACTIVE_COLOR_SELECT === "1nd-Color")
+     {
+       ACTIVE_COLOR_SELECT = "2nd-Color";
+       activeColorDiv_2.classList.add("active_indicator");
+       activeColorDiv_1.classList.remove("active_indicator");
+     }
+
```
Comment (Kully) [inline js/script.js:160]: Zap this blank line

File: main.css [hunk 4]
```diff
@@ -275,16 +275,29 @@ body {
...
-	width: var(--palette-width);
+	width: calc(var(--palette-width)/2.4);
 	height: var(--toolbar-height);
 	border: 1px solid #eef;
 	left: var(--toolbar-width);
 	border-radius: var(--border-radius);
 	margin-left: var(--palette-margin-right);
 	transform: translate(-2px, -1px);
 }
+#active-color-preview-2 {
```
Comment (Kully) [inline main.css:288]: There is a lot of overlap between `#active-color-preview-1` and `#active-color-preview-2`.

For the sake of reducing repeated code, can you define them like this?

```css

#active-color-preview-1 #active-color-preview-1 {
...
}

/* Then you can specify the things that are different below */
#active-color-preview-1 {
...
}
#active-color-preview-2 {
...
}

File: main.css [hunk 5]
```diff
@@ -275,16 +275,29 @@ body {
 	font-family: Menlo;
 }
 
-#active-color-preview {
+#active-color-preview-1 {
 	position: absolute;
-	width: var(--palette-width);
+	width: calc(var(--palette-width)/2.4);
```
Comment (Kully) [inline main.css:280]: Where does the 2.4 come from?

**Review by Kully — CHANGES_REQUESTED** [review]
Really good work.

There are a changes I've requested but should be good to go after that. :)

**Comment by aleksi100** [comment]
Thanks for good feedback, is it now better

**Comment by Kully** [comment]
> Thanks for good feedback, is it now better

Looks fantastic, thank you very much for addressing. :)

**Review by Kully — APPROVED** [review]

**Comment by Kully** [comment]
Merged 🎉🎉🎉 Thank you so much for your contribution.

**Pull request merged**